cmd/nodeagent/main.go

```diff
@@ -28,7 +28,7 @@ func main() {
 		log.Fatal(err)
 	}
 
-	n := NewNodeAgentManager(dynamicClient, nodeName, logEveryHeartbeat, ignoreHealthFromHeartbeat, time.Now)
+	n := NewNodeAgentManager(dynamicClient, nodeName, logEveryHeartbeat, ignoreHealthFromHeartbeat, time.Now, true)
```

Comment:
idea, maybe for a different PR: add an option (command line flag?) so user can turn it on or off (default should be on though)